src/olympia/devhub/templates/devhub/addons/activity.html

```diff
@@ -30,7 +30,7 @@ <h2>{{ _('Recent Activity for My Add-ons') }}</h2>
             {{ item }}
           </p>
           <p class="timestamp">
-          {% trans user=item.author_name,
+          {% trans user=item.user.name if item.user else generic_name,
```

Comment:
This is done in a couple of templates. If I cannot figure out a way to do this at a higher level (e.g., in the transformer), then maybe this duplication could be addressed via a jinja helper?

Reply:
Dealt with via `GenericMozillaUser`